lib/scale.rb

```diff
@@ -1,6 +1,37 @@
 class Scale
   attr_accessor :api_key, :callback_auth_key, :default_request_params, :logging
 
+  VALID_TASK_TYPES = [
+    "datacollection",
+    "categorization",
+    "comparison",
+    "annotation",
+    "polygonannotation",
+    "lineannotation",
+    "transcription",
+    "audiotranscription",
+    "pointannotation",
+    "segmentannotation"
+  ]
+
+  def method_missing(methodId, *args, &block)
+    str = methodId.id2name
+    match = /^create_([a-z_]+)_task$/.match(str)
+    if match
+      taskType = match[1].gsub(/[^a-z]/, '')
```

Comment:
What is this gsub for?

Reply:
Getting rid of underscores

Reply:
I think it could also be`/_/`, but if we were to add another character to the `[a-z_]+` clause two lines above, we'd need to update this.